packages/react/src/Restitution/SectionRestitutionRow.tsx

```diff
@@ -0,0 +1,32 @@
+import { PropsWithChildren } from "react";
+import { getComponentClassName } from "../Form/core";
+import { SectionRestitutionTitle } from "./SectionRestitutionTitle";
+
+const DEFAULT_CLASSNAME = "col col-sm-12 col-md-12 col-lg-12 col-xl-12";
+
+export type SectionRestitutionRowProps = {
+  title?: React.ReactNode;
+  classNameContainer?: string;
+  className?: string;
+  classModifier?: string;
+};
+
+export const SectionRestitutionRow = ({
+  title,
+  className = DEFAULT_CLASSNAME,
+  classNameContainer = "row af-restitution__content-left",
+  children = null,
+  classModifier,
+}: PropsWithChildren<SectionRestitutionRowProps>) => {
+  const componentClassName = getComponentClassName(
+    className,
+    classModifier,
+    DEFAULT_CLASSNAME,
+  );
+  return (
+    <div className={componentClassName}>
+      {title && <SectionRestitutionTitle title={title} />}
+      <div className={classNameContainer}>{children}</div>
```

Comment:
La `div` ne sert à rien si children est null ou undefined non ?
Si c'est bien le cas tu peux faire :

```suggestion
      {children && (<div className={classNameContainer}>{children}</div>)}
```

Reply:
Attention la condition que tu propose n'ai pas toujours vrais.
si je fait:
```javascript
<SectionRestitutionRow>
  {foo && <div>foo</div>
  {bar && <div>bar</dov>
</SectionRestitutionRow>
```
et que foo et bar valent false alors mon children cela un tableau [false, false] du coup la div cela présent alors que fonctionnement il n'y pas pas de children

Dans ce cas il faut passer par la fonction Children de react et faire comme ceci:
```javascript
{Children.toArray(childre).length !== 0 && ...}
```
La fonction Children.toArray, convertie le children en un tableau préfiltré dans null, false, undifined. 

note: si un des children est un Fragment il faudra le traiter autrement car c'est vue comme un children